dev-libs/spdlog/spdlog-1.9.2.ebuild

```diff
@@ -25,7 +25,7 @@ BDEPEND="
 	virtual/pkgconfig
 "
 DEPEND="
-	>=dev-libs/libfmt-6.1.2:=
+	>=dev-libs/libfmt-8.0.0:=
```

Comment:
Normally would need a revbump for a rdepend change, but given it's a build failure and there's a binding operator to prevent downgrading, it's all good (just wanted to make a note of it).